src/frontend/src/views/Homepage/index.tsx

```diff
@@ -22,49 +22,54 @@ import {
 
 export default function Homepage(): JSX.Element {
   return (
-    <Container>
-      <CardContainer>
-        <Card>
-          <Content>
-            <Left>
-              <Title>Welcome to Aspen!</Title>
-              <Main>
-                Aspen is a new tool that helps you manage, analyze, and share
-                your pathogen data for genomic epidemiology.
-              </Main>
-              <Details>
-                This is also where you’ll receive all of your sequencing results
-                from the COVIDTracker project at the CZ Biohub.
-              </Details>
-            </Left>
+    <>
+      <Head>
```

Comment:
Also only added the `Head` component here and moved `ConsensusGenomes` SVG into `Virus` styled component